client/TTTClient/TTTClient/Form1.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+using System.IO.Ports;
+using System.Threading;
+
+namespace TTTClient
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            InitializeComponent();
+
+            // For now: requires manual COM-port selection
+            SerialPort serial = new SerialPort("COM10", 115200);
```

Comment:
the same comment about magic nuber as in server.ino. And do not pass "COM10" as string literal or const - each time you will require to change it whrn device connetcted to diferent port or other laptop is used. 
You can pass it as CLI argument in CLI app or add textbox or combobox on GUI. Combobox is more userfriendly solution but you need to get active COM ports list to show.